src/cdev/commands/deploy.py

```diff
@@ -1,21 +1,17 @@
 from cdev.constructs.project import Project
-from cdev.default.output_manager import CdevOutputManager
 from cdev.cli.logger import set_global_logger_from_cli
 
-from core.commands.deploy_differences import execute_deployment
-
-
-def deploy_command_cli(args) -> None:
-    config = args[0]
-    set_global_logger_from_cli(config.loglevel)
-    deploy_command(args)
+from core.constructs.output_manager import OutputManager
 
+from core.commands.deploy_differences import execute_deployment
 
-def deploy_command(args) -> None:
 
-    output_manager = CdevOutputManager()
-    myProject = Project.instance()
+def deploy_command_cli(
+    project: Project, output_manager: OutputManager, **kwargs
+) -> None:
+    deploy_command(project, output_manager)
 
-    ws = myProject.get_current_environment().get_workspace()
 
+def deploy_command(project: Project, output_manager: OutputManager) -> None:
+    ws = project.get_current_environment().get_workspace()
     execute_deployment(ws, output_manager)
```

Comment:
did you ran pre-commit on all those files?
This one for example is lacking a empty line at the EOF, not a big deal.